src/lib/Scenes/Artwork/Components/CommercialButtons/InquiryModal.tsx

```diff
@@ -158,7 +159,9 @@ export const InquiryModal: React.FC<InquiryModalProps> = ({ artwork, ...props })
           resetAndExit()
         }}
         rightButtonText="Send"
-        rightButtonDisabled={state.inquiryQuestions.length === 0}
+        rightButtonDisabled={
+          (state.inquiryQuestions.length === 0 && !!questions.length) || (!questions.length && !state.message)
```

Comment:
I'm trying to work through this in my head, but I think this logic isn't correct. 

I think it's supposed to be as follows:

If no question is checked and the message is empty, disable the button.

Reply:
There are three factors considered here

1. Are there questions
2. Are any questions selected
3. Is there a message

In the situation that there are no questions we expect that there must be a message. If there's no message we expect that there must be questions selected. I think we can leverage those two facts to simplify the overall logic to just two factors.

```suggestion
          (state.inquiryQuestions.length === 0 && !state.message)
```

Reply:
I think my brain might still not be in full working order; this took me a minute.

Yeah, you're right. Nice, @zephraph! 💪